### ISSUE TITLE
Adding `onEventResize` functionality end-date offset issue

### ISSUE THREAD

**Issue by scott-wooden** [cutterbl/react-big-calendar-with-timezones#1]
Hi cutterbl firstly this is an awesome solution to the timezone issue faced within BigCalendar.

I've got an issue with resizing the end date. By adding the `onEventResize` props when we resize the end accessor the value is +1 the date that you wish to select. Happens on any timezone instance. 

I've managed to replicate it within your codepen sketch shown below:
![mHIgx6mk2z](https://user-images.githubusercontent.com/61710970/78781537-59fd8c00-7998-11ea-8ace-a616881a852f.gif)

I've hit a brick wall with this. Don't suppose you have any insights as to what might be happening?

Thanks!

**Comment by cutterbl** [comment]
Yeah, my initial stuff did basic TZ stuff fine. I have since found out that some of RBC's underlying logic still doesn't play nice, regardless of localizer. I submitted an Issue on the core project, where you can [read some of my findings](https://github.com/jquense/react-big-calendar/issues/1478).
